### PR TITLE
Add polling feature for new images

### PR DESCRIPTION
**Pull request by benpbenp**
- needs testing with multiple users!
- maybe 5 seconds is too aggressive

### CODE REVIEW

File: frontend/index.jsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ class App extends React.Component {
       aesKey: null,
       tab: 'my'
     }
+    this.readSubscribersImages = this.readSubscribersImages.bind(this);
```
Comment (tjwelde) [inline frontend/index.jsx]: This seems unnecessary? You might want to do that at the exact place you need that ;)

Comment (benpbenp) [reply frontend/index.jsx]: In its defense... I think this pattern is used in a lot of React projects. It kind of puts the boilerplate in one place, and also it seems unlikely to me that we'd need to use this method without the binding.

Comment (elmariachi111) [reply frontend/index.jsx]: this looks quite weird for me too. I mean, "this.readSubscribersImages" should have bound "this" within its constructor closure anyway. If the newly bound "this" is sth different that the original "this" this method should be rewritten :D

**Comment by benpbenp** [comment]
Removed stage-2 preset and just using transform-object-rest-spread-plugin plus other minor edits

**Comment by benpbenp** [comment]
@goldquest and I discussed this and I think the next step is to present a notification at the top when there are new images to load, which can be clicked to actually load them, rather than loading them immediately.

**Review by tjwelde — APPROVED** [review]
LGTM

**Pull request merged**